src/applications/caregivers/containers/IntroductionPage.jsx

```diff
@@ -14,15 +14,26 @@ import { focusElement } from 'platform/utilities/ui';
 
 import CaregiversPrivacyActStatement from '../components/IntroductionPage/CaregiversPrivacyActStatement';
 import ProcessTimeline from '../components/IntroductionPage/ProcessTimeline';
-import { appTitle, appSubtitle, appIntro } from '../definitions/content';
-
-export const IntroductionPage = ({
-  route,
-  router,
-  formData,
-  setFormData,
-  useFacilitiesAPI,
-}) => {
+import content from '../locales/en/content.json';
+
+export const IntroductionPage = props => {
+  const { route, router, formData, setFormData, useFacilitiesAPI } = props;
+
+  const startForm = useCallback(
+    () => {
+      recordEvent({ event: 'caregivers-10-10cg-start-form' });
+      const { pageList } = route;
+      return router.push(pageList[1].path);
+    },
+    [route, router],
+  );
+
+  const startBtn = (
```

Comment:
Lets pull this out of the markup and into a variable, since its repeated on the page.